pkgs/by-name/li/libvmi/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  autoconf-archive,
+  bison,
+  flex,
+  glib,
+  pkg-config,
+  json_c,
+  libvirt,
+
+  withVMIFS ? true,
+  fuse,
+
+  legacyKVM ? false,
+  libkvmi,
+
+  xenSupport ? true,
+  xen-slim,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libvmi";
+  version = "0.14.0-unstable-2024-07-25";
+  libVersion = "0.0.15";
```

Comment:
This should be in a let in or if required to be exposed to other packages be in passthru